hack/images/ci/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN /google-cloud-sdk/bin/gcloud components update
 ##                               KIND STAGE                                   ##
 ################################################################################
 FROM ${GOLANG_IMAGE} as kind
-RUN go get -u sigs.k8s.io/kind
+RUN GO111MODULE="on" go get -u sigs.k8s.io/kind@v0.3.0
```

Comment:
/hold

Hi @yastij,

Won't this update the project's `go.mod` and `go.sum` that are then persisted into the final container?

Reply:
Maybe update the `RUN` line to `cd / && ...` to ensure it doesn't execute in the project directory.

Reply:
@akutz - oh right, good catch !